pandas/plotting/_misc.py

```diff
@@ -470,19 +474,21 @@ def lag_plot(series: Series, lag: int = 1, ax: Axes | None = None, **kwds) -> Ax
 
     Parameters
     ----------
-    series : Time series
-    lag : lag of the scatter plot, default 1
+    series : Series
+        The time series to visualize.
+    lag : int, optional
```

Comment:
```suggestion
    lag : int, default 1
```

We use `optional` when the default values is `None`, meaning not provided (as opposed a `None` value that is used, for example to replace a value by `None`...).

Reply:
Fixed, thanks.